.gitmodules

```diff
@@ -4,3 +4,9 @@
 [submodule "lib/openzeppelin-contracts"]
 	path = lib/openzeppelin-contracts
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts
+[submodule "forge-std"]
```

Comment:
What's the reason behind adding this? We already have `forge-std` in `lib/forge-std`. Why did you add another one to the base directory?

Reply:
This is a mistake, has been removed now.